packages/terra-menu/src/_MenuContent.jsx

```diff
@@ -248,39 +254,44 @@ class MenuContent extends React.Component {
     }
 
     const backIcon = <IconLeft />;
-    const backButton = this.props.index > 0 ? (
-      // eslint-disable-next-line jsx-a11y/no-static-element-interactions
-      <div
-        role="button"
-        onClick={this.props.onRequestBack}
-        onKeyDown={this.onKeyDownBackButton}
-        tabIndex="0"
-        aria-label={backBtnText}
-      >
-        <Arrange
-          align="center"
-          fitStart={(
+    let header = <div />;
+
+    if (this.props.index > 0) {
+      header = (
+        <div
+          role="button"
+          onClick={this.props.onRequestBack}
+          onKeyDown={this.onKeyDownBackButton}
+          tabIndex="0"
+          aria-label={backBtnText}
+        >
+          <div className={cx('header-container')}>
             <div className={cx('header-button')}>
               {backIcon}
             </div>
-          )}
-          fill={<h1 className={cx('header-title')}>{this.props.title}</h1>}
-        />
-      </div>
-    ) : <div />;
+            <h1 className={cx('header-title')}>{this.props.title}</h1>
+          </div>
+        </div>
+      );
+    } else if (this.props.headerTitle.length > 0) {
+      header = (
+        <h1 className={cx(['header-title', 'main-header-title'])}>{this.props.headerTitle}</h1>
+      );
+    }
 
     return (
       <Arrange
         className={cx('header')}
         fitEnd={closeButton}
-        fill={backButton}
+        fill={header}
         align="center"
       />
     );
   }
 
   render() {
     let index = -1;
+    let isMainMenuHeader;
```

Comment:
I think this variable would make more sense as `shouldDisplayMainMenuHeader`

Reply:
updated here 16ed11f47288f7ab7e65f9410e955857d99c80f1